backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java

```diff
@@ -124,7 +147,169 @@ public String caseDefinition(Definition def) {
+        if (ownedReferenceSubsetting != null) {
+            builder.appendSpaceIfNeeded().append(LabelConstants.REFERENCES);
+            if (ownedReferenceSubsetting.getReferencedFeature() != null) {
+                builder.appendSpaceIfNeeded().append(getDeresolvableName(ownedReferenceSubsetting.getReferencedFeature(), element));
+            }
+        }
+
+    }
+
+    /**
+     * @param builder
+     * @param ownedTyping
+     * @param element
+     */
+    private void appendFeatureTyping(Appender builder, EList<FeatureTyping> ownedTyping, Element element) {
+        if (!ownedTyping.isEmpty()) {
+            builder.appendSpaceIfNeeded().append(LabelConstants.DEFINED_BY);
+            builder.appendSpaceIfNeeded().append(ownedTyping.stream()
+                    .filter(ft -> ft.getType() != null)
+                    .map(ft -> Tuples.of(ft, ft.getType()))
+                    .filter(t -> t.getT2() != null)
+                    .map(t -> {
+                        String name = getDeresolvableName(t.getT2(), element);
+
+                        if (t.getT1() instanceof ConjugatedPortTyping) {
+                            name = LabelConstants.CONJUGATED + name;
+                        }
+                        return name;
+                    }
+
+                    )
+                    .collect(Collectors.joining(", ")));
+        }
+    }
+
+    private void appendMultiplicityPart(Appender builder, Feature feature) {
+        MultiplicityRange multiplicity = feature.getOwnedElement().stream()
+                .filter(MultiplicityRange.class::isInstance)
+                .map(MultiplicityRange.class::cast)
+                .findFirst()
+                .orElse(null);
+
+        if (multiplicity != null) {
+            String expression = multiplicity.getOwnedElement().stream()
+                    .filter(element -> element instanceof LiteralExpression || element instanceof FeatureReferenceExpression)
+                    .map(element -> {
+                        String exp = null;
+                        if (element instanceof LiteralExpression lit) {
+                            exp = caseLiteralExpression(lit);
+                        } else if (element instanceof FeatureReferenceExpression featureReferenceExp) {
+                            exp = getDeresolvableName(featureReferenceExp, feature);
+                        }
+                        return exp;
+                    })
+                    .filter(Objects::nonNull)
+                    .collect(Collectors.joining(".."));
+
+            if (!expression.isEmpty()) {
+                builder.appendSpaceIfNeeded().append(LabelConstants.OPEN_BRACKET)
+                        .append(expression)
+                        .append(LabelConstants.CLOSE_BRACKET);
+            }
+
+            boolean isOrdered = multiplicity.isIsOrdered();
+            boolean isUnique = multiplicity.isIsUnique();
+
+            if (isOrdered) {
```

Comment:
I think this is not correct.

What about if _isUnique_ = _true_ and _isOrdered_ = false ?

Reply:
done